compiler/circle-inspect/src/Dump.cpp

```diff
@@ -240,3 +240,38 @@ void DumpConstants::run(std::ostream &os, const circle::Model *model, const std:
 }
 
 } // namespace circleinspect
+
+namespace circleinspect
+{
+
+void DumpTensorShape::run(std::ostream &os, const circle::Model *model,
+                          const std::vector<char> *data)
+{
+  mio::circle::Reader reader(model, data);
+
+  const uint32_t subgraph_size = reader.num_subgraph();
+
+  for (uint32_t g = 0; g < subgraph_size; g++)
+  {
+    reader.select_subgraph(g);
+    auto tensors = reader.tensors();
+
+    for (uint32_t i = 0; i < tensors->size(); ++i)
+    {
+      const auto tensor = tensors->Get(i);
+      auto shape = tensor->shape_signature() ? tensor->shape_signature() : tensor->shape();
+      os << reader.tensor_name(tensor) << " ";
+      for (int8_t i = 0; i < shape->size(); i++)
```

Comment:
It's not a must, but most machines do calculations in 32-bit units, so there's probably no need to reduce it to 8-bit in `for loop`.

```suggestion
      for (uint32_t i = 0; i < shape->size(); i++)
```

Reply:
> It's not a must, but most machines do calculations in 32-bit units, so there's probably no need to reduce it to 8-bit in `for loop`.

I'll update it.